#!/usr/bin/env python
PACKAGE = "hi_controllers"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("kp_position",    double_t,    0,  "Kp_position",    0,    0,   300)
gen.add("kd_position",    double_t,    1,  "Kd_position",    5,    0,   100)
gen.add("kp_big_stance",    double_t,    1,  "kp_big_stance",    20,    0,   100)
gen.add("kp_big_swing",    double_t,    1,  "kp_big_swing",    20,    0,   100)
gen.add("kd_big",    double_t,    1,  "kd_big",    5,    0,   20)

gen.add("kp_small_stance",    double_t,    1,  "kp_small_stance",    20,    0,   100)
gen.add("kp_small_swing",    double_t,    1,  "kp_small_swing",    20,    0,   100)
gen.add("kd_small",    double_t,    1,  "kd_small",    5,    0,   20)

gen.add("kd_feet",    double_t,    1,  "kd_feet",    1,    0,   20)

exit(gen.generate(PACKAGE, "hi_controllers", "Tutorials")) 